packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1126,6 +1141,49 @@ export function mixinStashedClient<
 	};
 }
 
+export function mixinHandle<
+	TChannelFactory extends IChannelFactory,
+	TOperation extends BaseOperation,
+	TState extends DDSFuzzTestState<TChannelFactory>,
+>(
+	model: DDSFuzzModel<TChannelFactory, TOperation, TState>,
+	options: DDSFuzzSuiteOptions,
+): DDSFuzzModel<TChannelFactory, TOperation | HandleCreated, TState> {
+	if (options.handles === false) {
+		return model as DDSFuzzModel<TChannelFactory, TOperation | HandleCreated, TState>;
+	}
+
+	const generatorFactory: () => AsyncGenerator<TOperation | HandleCreated, TState> = () => {
+		const baseGenerator = model.generatorFactory();
+		return async (state): Promise<TOperation | HandleCreated | typeof done> => {
+			const baseOp = baseGenerator(state);
```

Comment:
this doesn't look right. there is no need to generate the base op until it is needed, so just return  baseGenerator(state) if we aren't returning a handle op